packages/frameworks/src/frameworks.ts

```diff
@@ -251,8 +251,8 @@ export const frameworks = [
     ],
     defaultHeaders: [
       {
-        source: '^/build/(.*)$',
-        regex: '^/build/(.*)$',
+        source: '/build/(.*)',
+        regex: '/build/(.*)',
```

Comment:
```suggestion
        source: '/build/(.*)',
        regex: '^/build/(.*)$',
```

This should stay for the `regex`, but it's not required for `source`. Or was their a specific reason for why you removed it?

Reply:
Yes, I removed it because it caused a double regex in the final output, even though `routes-manifest.json` ended up being correct:
 
<img width="653" alt="CleanShot 2021-11-25 at 11 25 01@2x" src="https://user-images.githubusercontent.com/6170607/143427739-f7a7743f-0e6f-434e-9ded-2826f3f973fd.png">

<img width="591" alt="CleanShot 2021-11-25 at 11 48 15@2x" src="https://user-images.githubusercontent.com/6170607/143427789-ae3ecefe-6e08-4da1-a118-c1bac76308f8.png">

![CleanShot 2021-11-25 at 11 27 41@2x](https://user-images.githubusercontent.com/6170607/143427811-3043636d-4c56-4072-8bec-4a8818037276.png)

So either the File System API (not `vercel build`) is behaving incorrectly, or the file you commented on wasn't correct before my change.

Reply:
I see, I'll investigate later on about this one.